src/api/integrations/event/webhook/webhook.controller.ts

```
@@ -74,10 +74,20 @@ export class WebhookController extends EventController implements EventControlle
 
     const webhookConfig = configService.get<Webhook>('WEBHOOK');
     const webhookLocal = instance?.events;
-    const webhookHeaders = instance?.headers;
+    const webhookHeaders = { ...((instance?.headers as Record<string, string>) || {}) };
+
+    if (webhookHeaders && 'jwt_key' in webhookHeaders) {
+      const jwtKey = webhookHeaders['jwt_key'];
+      const jwtToken = this.generateJwtToken(jwtKey);
+      webhookHeaders['Authorization'] = `Bearer ${jwtToken}`;
+
+      delete webhookHeaders['jwt_key'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)